features/us20_borrower_views_deed.feature

```diff
@@ -17,7 +17,9 @@ Scenario: Borrower enters a valid deed reference
          - e-MD ref
 
   When I navigate to the borrower frontend "/searchdeed" page
-  And I search for the created deed
+  When I retrieve the deed id
+  And I retrieve the unique user id using the URL
```

Comment:
Would be tempted to change user id to borrower token just to keep some consistency with the language we are using to refer to things to avoid confusion.